posthog/tasks/usage_report.py

```diff
@@ -580,7 +580,7 @@ def get_teams_with_hogql_metric(
         FROM clusterAllReplicas({CLICKHOUSE_CLUSTER}, system.query_log)
         WHERE (type = 'QueryFinish' OR type = 'ExceptionWhileProcessing')
           AND is_initial_query = 1
-          AND query_type IN (%(query_types)s)
+          AND (%(query_types)s IS NULL OR query_type IN (%(query_types)s))
```

Comment:
Nit: This generates some strange code like:

```sql
    AND (NULL IS NULL
         OR query_type IN (NULL))
```

and

```sql
    AND (['EventsQuery'] IS NULL
         OR query_type IN (['EventsQuery']))
```

Since `query_types` is a python variable, wouldn't it be better to rewrite the entire clause based on its contents?

Reply:
yeah let me rewrite that, thanks